src/main/java/org/folio/client/feign/RequestStorageClient.java

```diff
@@ -17,4 +20,6 @@ public interface RequestStorageClient {
   @PutMapping("/{requestId}")
   Request updateRequest(@PathVariable String requestId, @RequestBody Request request);
 
+  @GetMapping(params = "query")
+  List<Request> getRequestsByQuery(@RequestParam("query") String query);
```

Comment:
See if we can use Request Queue API in mod-circulation:
https://s3.amazonaws.com/foliodocs/api/mod-circulation/r/circulation.html#